2021-11-23 18:22:11,317 - DEBUG - Popen(['git', 'init'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:22:11,384 - DEBUG - Failed checking if running in CYGWIN due to: FileNotFoundError(2, '系统找不到指定的文件。', None, 2, None)
2021-11-23 18:22:11,385 - DEBUG - Establish git sucess!
2021-11-23 18:22:11,386 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:22:11,434 - DEBUG - Popen(['git', 'remote', 'rm', 'origin'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:22:11,489 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:22:11,541 - DEBUG - Get remote sucess!
2021-11-23 18:22:36,745 - DEBUG - Popen(['git', 'cat-file', '--batch-check'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=<valid stream>)
2021-11-23 18:22:36,797 - DEBUG - Popen(['git', 'cat-file', '--batch'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=<valid stream>)
2021-11-23 18:22:40,236 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:22:40,236 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:22:40,237 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:22:40,237 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:25:54,574 - DEBUG - Popen(['git', 'init'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:25:54,645 - DEBUG - Failed checking if running in CYGWIN due to: FileNotFoundError(2, '系统找不到指定的文件。', None, 2, None)
2021-11-23 18:25:54,647 - DEBUG - Establish git sucess!
2021-11-23 18:25:54,648 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:25:54,701 - DEBUG - Popen(['git', 'remote', 'rm', 'origin'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:25:54,758 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:25:54,810 - DEBUG - Get remote sucess!
2021-11-23 18:26:04,530 - DEBUG - Popen(['git', 'cat-file', '--batch-check'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=<valid stream>)
2021-11-23 18:26:04,592 - DEBUG - Popen(['git', 'cat-file', '--batch'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=<valid stream>)
2021-11-23 18:26:05,415 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:26:05,416 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:27:03,506 - DEBUG - Popen(['git', 'init'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:27:03,575 - DEBUG - Failed checking if running in CYGWIN due to: FileNotFoundError(2, '系统找不到指定的文件。', None, 2, None)
2021-11-23 18:27:03,578 - DEBUG - Establish git sucess!
2021-11-23 18:27:03,579 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:27:03,643 - DEBUG - Popen(['git', 'remote', 'rm', 'origin'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:27:03,699 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:27:03,750 - DEBUG - Get remote sucess!
2021-11-23 18:27:30,282 - DEBUG - Popen(['git', 'init'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:27:30,354 - DEBUG - Failed checking if running in CYGWIN due to: FileNotFoundError(2, '系统找不到指定的文件。', None, 2, None)
2021-11-23 18:27:30,356 - DEBUG - Establish git sucess!
2021-11-23 18:27:30,356 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:27:30,434 - DEBUG - Popen(['git', 'remote', 'rm', 'origin'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:27:30,489 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:27:30,541 - DEBUG - Get remote sucess!
2021-11-23 18:27:35,959 - DEBUG - Popen(['git', 'cat-file', '--batch-check'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=<valid stream>)
2021-11-23 18:27:36,014 - DEBUG - Popen(['git', 'cat-file', '--batch'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=<valid stream>)
2021-11-23 18:27:40,472 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:27:40,473 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:27:40,473 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:27:40,473 - INFO - Ignored error after process had died: OSError(9, '句柄无效。', None, 6, None)
2021-11-23 18:28:38,327 - DEBUG - Popen(['git', 'init'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:28:38,396 - DEBUG - Failed checking if running in CYGWIN due to: FileNotFoundError(2, '系统找不到指定的文件。', None, 2, None)
2021-11-23 18:28:38,398 - DEBUG - Establish git sucess!
2021-11-23 18:28:38,398 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:28:38,455 - DEBUG - Popen(['git', 'remote', 'rm', 'origin'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:28:38,512 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:28:38,561 - DEBUG - Get remote sucess!
2021-11-23 18:28:52,828 - DEBUG - Popen(['git', 'cat-file', '--batch-check'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=<valid stream>)
2021-11-23 18:28:52,881 - DEBUG - Popen(['git', 'cat-file', '--batch'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=<valid stream>)
2021-11-23 18:29:56,548 - DEBUG - Popen(['git', 'init'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:29:56,619 - DEBUG - Failed checking if running in CYGWIN due to: FileNotFoundError(2, '系统找不到指定的文件。', None, 2, None)
2021-11-23 18:29:56,621 - DEBUG - Establish git sucess!
2021-11-23 18:29:56,621 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:29:56,681 - DEBUG - Popen(['git', 'remote', 'rm', 'origin'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:29:56,738 - DEBUG - Popen(['git', 'remote', 'add', 'origin', 'git@github.com:c369578336/test.git'], cwd=D:\data\project\test, universal_newlines=False, shell=None, istream=None)
2021-11-23 18:29:56,787 - DEBUG - Get remote sucess!
